### ISSUE TITLE
Screenreader always repeats name of picker after every single item

### ISSUE THREAD

**Issue by weinand** [microsoft/vscode#93928]
Refs #93648 
Orca 3.28.0 on Ubuntu 18.04.4 LTS

- press F1
- use cursor down key to step through items

Screen reader reads "File Picker" after every item.
Is this really necessary?

**Comment by bpasero** [comment]
@isidorn thoughts? easy to remove. we used to put the picker for each item already in the old impl

**Comment by weinand** [comment]
I have no experience with screen readers. I just report what I noticed.
Someone who has to rely on a screen reader should test this and provide feedback.

**Comment by isidorn** [comment]
If I press F1 for the the screen reader does not read "File Picker" but reads "commands picker" at the end. If I press ctrl + p to get the File Picker than "File Picker" is read. Which is correct.
I think this is fine since we read it at the end and most screen reader users just stop the announcment and go to the next item.
Though let's see what @jvesouza has to say about this

**Comment by jvesouza** [comment]
@weinand @isidorn 
When I press f1 and press down arrow, orca reads in the end of each command commands picker, and when I press ctrl+p orca reads in the end of command recently opened.
the text in the end is in my opinion not relevant information and could be deleted, as long as the effort was not too much. But at least for me, it's not much of a problem because I can quickly silence the orca pressing the ctrl key or move on to the next item.

**Comment by isidorn** [comment]
@jvesouza thanks for feedback.
So I leave this up to @bpasero to decide as the owner of the feature
My suggestion is to remove the ending.

**Comment by bpasero** [comment]
Removed announcement of the picker you are in from all of quick open items.

**Comment by jvesouza** [comment]
It was in my opinion very good. Less information to be read by the screen reader.
Thanks.

**Comment by isidorn** [comment]
@jvesouza great, thanks for letting us know. Adding verified label.
